api/http/management_filetransfer.go

```diff
@@ -516,7 +516,7 @@ func (h ManagementController) UploadFile(c *gin.Context) {
 		request.GID = &nGID
 	}
 	if mode := c.Request.Form.Get(fieldUploadMode); mode != "" {
-		v, _ := strconv.Atoi(mode)
+		v, _ := strconv.ParseInt(mode, 8, 64)
```

Comment:
are we sure we are ignoring error here?

Reply:
yes, default is zero which is the null value of int